packages/core/src/components/portal/portal.tsx

```diff
@@ -67,8 +76,11 @@ export class Portal extends React.Component<IPortalProps, IPortalState> {
     }
 
     public componentDidMount() {
+        if (!this.props.container) {
```

Comment:
🤔 this could actually be a serious problem... `cDM` is only called once in the lifecycle and if `container` doesn't exist at that time then _the portal will never mount_. i think we need to address this case before we can merge.

Reply:
https://github.com/palantir/blueprint/blob/46df44d90aa13e06beffe5a6764027630baa8349/packages/core/src/components/portal/portal.tsx#L71
If `container` not exists (`typeof document === 'undefined'`, `render()` method will return `null`. It's nothing changed than before.